reportportal_client/core/worker.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, cmd_queue, data_queue):
     def _command_get(self):
         """Get control command from the control queue."""
         try:
-            cmd = self._cmd_queue.get_nowait()
+            cmd = self._cmd_queue.get(timeout=0.1)
```

Comment:
I don't like the idea of two queues: If command arrives into `_cmd_queue` while interpreter waits for `_data_queue` it will be a redundant delay of 100 milliseconds in worse case and vice versa. 

I already implemented a single-queue solution here: https://github.com/reportportal/client-Python/tree/cpu-consumption-fix
But you are free to suggest your own implementation.